Guide/form.markdown

```diff
@@ -5,8 +5,346 @@
 
 ## Introduction
 
+In IHP Forms are an essential way to interact with your application. Dealing with a lot of form markup can quickly become complex because of the need to deal with consistent styling and especially when dealing with lots of validation. IHP provides helpers to generate form markup to help you deal with the complexity.
+
+By default forms in IHP follows the class names used by Bootstrap 4. Therefore the forms work with Bootstrap 4 out of the box. Of course the default form generation can be customized to support other CSS frameworks.
+
+Unless javascript helpers have been deactivated, your form will be submitted using AJAX and TurboLinks instead of a real browser based form submission.
```

Comment:
Please clarify the following:
- How can they be deactivated (link)?
- What happens when JS helpers are used, but JS is disabled? Does it fall back to HTML form submission gracefully, or does it break?

Reply:
Good points :) Also added a section for this.